code/modules/research/nanites/nanite_programs/utility.dm

```diff
@@ -258,6 +258,32 @@
 			SEND_SIGNAL(infectee, COMSIG_NANITE_SYNC, nanites)
 			infectee.investigate_log("was infected by spreading nanites by [key_name(host_mob)] at [AREACOORD(infectee)].", INVESTIGATE_NANITES)
 
+/datum/nanite_program/triggered/nanite_sting
+	name = "Nanite Sting"
+	desc = "Stings a random non-host around the host with a barely-visible cluster of nanites, making them a new host. The sting might be felt."
+	trigger_cost = 5
+	trigger_cooldown = 30
+	rogue_types = list(/datum/nanite_program/glitch, /datum/nanite_program/toxic)
+
+/datum/nanite_program/triggered/nanite_sting/trigger()
+	if(!..())
+		return
+	var/list/mob/living/target_hosts = list()
+	for(var/mob/living/L in oview(1, host_mob))
+		if(!(L.mob_biotypes & (MOB_ORGANIC|MOB_UNDEAD)))
+			continue
+		target_hosts += L
+	if(!target_hosts.len)
+		consume_nanites(-5)
+		return
+	var/mob/living/infectee = pick(target_hosts)
+	if(prob(100 - (infectee.get_permeability_protection() * 100)))
+		//just like with Infective Exo-Locomotion, this can take over existing nanites.
+		infectee.AddComponent(/datum/component/nanites, 5)
+		SEND_SIGNAL(infectee, COMSIG_NANITE_SYNC, nanites)
+		infectee.investigate_log("was infected by a nanite cluster by [key_name(host_mob)] at [AREACOORD(infectee)].", INVESTIGATE_NANITES)
+		to_chat(infectee, "<span class='warning'>You feel a tiny prick.</span>")
```

Comment:
there's not a chance they notice here, they always notice.

Reply:
So what will it be? Will people notice? I'd say a good 60% would. (that is, if you're referring to the tiny prick; otherwise, I'm not sure what you're talking about)

Reply:
I'm saying that "Targets of this program have a chance to notice that they got stung." is not accurate; this code will always send a message to the infectee

Reply:
So you're saying I should correct that. Correct?